adapter/noop2/noop2.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Istio Authors.
```

Comment:
I'm still unclear on why we're splitting `Configure` from `Build`. Is the goal that we do something like:

```go
builder := noop2Builder{} // we need an interface method to construct builders BTW
handlers := []Handler{}
for _, cfg := range handlers {
    builder.Configure(cfg)
    handlers = append(handlers, builder.Build())
}
```

If that's the goal, I think we'd be better off with
```go
builder := noop2Builder{} // we need an interface method to construct builders BTW
handlers := []Handler{}
for _, cfg := range handlers {
    handlers = append(handlers, builder.Build(cfg))
}
```
or even change `Build` to `BuildSample` if we need to support multiple kinds.